build.sbt

```diff
@@ -72,7 +75,7 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
 val compile = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-play-26" % bootstrapPlayVersion,
+  "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % bootstrapFrontendVersion,
   "uk.gov.hmrc" %% "govuk-template" % govTemplateVersion,
   "uk.gov.hmrc" %% "play-language" % "4.2.0-play-26",
```

Comment:
Is there any reason for not using the playLanguageVersion variable defined above (and in particular for using an older version here)?